src/Avatar.js

```diff
@@ -27,6 +27,7 @@ export default class Avatar extends React.Component {
         <View style={[styles[this.props.position].container, this.props.containerStyle[this.props.position]]}>
           <GiftedAvatar
             avatarStyle={StyleSheet.flatten([styles[this.props.position].image, this.props.imageStyle[this.props.position]])}
+            onPress={this.props.onAvatarPress}
```

Comment:
We probably don't want `onPress` here because this is actually a blank avatar that isn't visible.